app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    instance_list = []
+    for human in people:
+        instance = Person(human["name"], human["age"])
+        instance_list.append(instance)
+
+    for instance in instance_list:
+        for human in people:
+            if human["name"] == instance.name:
+                if "wife" not in instance.__dict__ and \
```

Comment:
what is the purpose of this check ? you dont have another chance in your code to add wife/husband in the __dict__ so you probably can delete it?